src/services/invitation/service.ts

```diff
@@ -130,7 +130,7 @@ export class InvitationService {
     // invitations to memberships is triggered on register: no actor available
     const { invitationRepository, itemMembershipRepository } = repositories;
     const invitations = await invitationRepository.find({
-      where: { email: member.email },
+      where: { email: member.email.toLowerCase() },
```

Comment:
should the email have been lowercase from the beginning (on registration) as well?

Reply:
It is being lowercased on registration [see in the `post` method of the `MemberService`](https://github.com/graasp/graasp/blob/391e0479aabf92a64f2363fe1b86c6ffacb311f3/src/services/member/service.ts#L36)